### ISSUE TITLE
ENORESTARGET error installing through bower

### ISSUE THREAD

**Issue by ahmadazimi** [babakhani/pwt.datepicker#40]
Recently I've tried to update my bower libraries. But there is a problem when bower try to update persian-datepicker (ENORESTARGET).

I tried to install a fresh one, but that error still remains:

```
bower install persian-datepicker --save
bower persian-datepicker#*      cached git://github.com/babakhani/pwt.datepicker.git#0.4.0
bower persian-datepicker#*    validate 0.4.0 against git://github.com/babakhani/pwt.datepicker.git#*
bower persian-date#~1.7.0   not-cached git://github.com/babakhani/PersianDate.git#~1.7.0
bower persian-date#~1.7.0      resolve git://github.com/babakhani/PersianDate.git#~1.7.0
bower persian-date#~1.7.0 ENORESTARGET No tag found that was able to satisfy ~1.7.0

Additional error details:
Available versions: 0.1.8
```

**Comment by babakhani** [comment]
This case will be solved in next release 0.4.1 as soon as possible.
Thanks for reporting this issue.

**Comment by ahmadazimi** [comment]
Thank you.
I'm keep waiting for that.

**Comment by babakhani** [comment]
Fix in tag 0.4.4

**Comment by ahmadazimi** [comment]
Thank you, I checked it and everything is okay.
